src/app/client/projects/desktop/src/app/modules/offline/components/library-filters/library-filters.component.ts

```diff
@@ -61,9 +70,26 @@ export class LibraryFiltersComponent implements OnInit {
                     break;
             }
         });
+
+        this.libraryFilters.emit([]);
     }
 
     getSelectedFilters(event, type) {
+        this.filterQueryParam.board = ['CBSE'];
+
+        if (type === 'medium') {
+            this.filterQueryParam.medium = [event.data.text];
+        } else if (type === 'class') {
+            this.filterQueryParam.gradeLevel = [event.data.text];
+        }
+
+        this.filterQueryParam['appliedFilters'] = true;
+        this.router.navigate([], { relativeTo: this.activatedRoute.parent, queryParams: this.filterQueryParam });
```

Comment:
is this component emits event when filter changes or routes to fetch relevant content